src/js/angular/rest/sparql.rest.service.js

```diff
@@ -0,0 +1,52 @@
+angular
+    .module('graphdb.framework.rest.sparql.service', [])
+    .factory('SparqlRestService', SparqlRestService);
+
+SparqlRestService.$inject = ['$http', '$repositories'];
+
+const SAVED_QUERIES_ENDPOINT = 'rest/sparql/saved-queries';
+const REPOSITORIES_ENDPOINT = 'repositories/';
+
+function SparqlRestService($http, $repositories) {
+    return {
+        getRepositorySize,
+        getSavedQueries,
+        getSavedQuery,
+        addKnownPrefixes,
+        editSavedQuery,
+        deleteSavedQuery,
+        addNewSavedQuery,
+    };
+
+    function getRepositorySize() {
+        return $http.get(`${REPOSITORIES_ENDPOINT}${$repositories.getActiveRepository()}/size`);
```

Comment:
Isn't this already present in `RepositoriesRestService` ?

Reply:
RepositoriesRestService is the rest client for the GDB `rest/repositories` endpoint and this is the RDF4J's `repositories` endpoint, which reminds me that it should be in the RDF4JRepositoriesRestService.

Reply:
Done